test/transcoding/transcoding_test.exs

```diff
@@ -1,17 +1,33 @@
 defmodule TranscodingTest do
-  alias Membrane.Pipeline
+  import Membrane.Testing.Assertions
+  alias Membrane.Testing.Pipeline
```

Comment:
Aliases are not ordered alphabetically. (not always needed, but if you are touching the file that's a nice code quality touch, at least for me)